src/CommunityToolkit.Maui.UnitTests/Mocks/AnimationExtensions.cs

```diff
@@ -74,6 +74,34 @@ public override async void Start()
 			}
 
 			public override void Stop() => cancellationTokenSource?.Cancel();
+
+			bool isDisposed;
+
+			public void Dispose()
+			{
+				Dispose(true);
+				GC.SuppressFinalize(this);
+			}
+
+			protected virtual void Dispose(bool disposing)
+			{
+				if (isDisposed)
+				{
+					return;
+				}
+
+				if (disposing)
+				{
+					cancellationTokenSource?.Dispose();
+				}
+
+				isDisposed = true;
+			}
+
+			~AsyncTicker()
```

Comment:
From my memory. We should just use this if we handle unmanaged resources. This can lead to bad things.

[Here's a reference](https://pvs-studio.com/en/blog/posts/csharp/0437/)

Reply:
I recommend keeping the finalizer. It's basically a fail-safe in case the consumer forgets to dispose an IDisposable. 

But, this is a private nested class inside of a UnitTest project, so we don't need to worry too much about this.